src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -179,7 +222,30 @@ public void updateObservablePersonList(Predicate<Person> predicate) {
     @Override
     public void updateObservablePersonList() {
         filteredPersons.setPredicate(PREDICATE_SHOW_ALL_PERSONS);
-        observablePersons.setComparator(COMPARATOR_CONTACT_INDEX);
+        observablePersons.setComparator(COMPARATOR_CONTACT_INDEX_PERSON);
+    }
+
+    @Override
+    public ObservableList<Recommendation> getObservableRecommendationList() {
+        return observableRecommendations;
+    }
+
+    @Override
+    public void updateObservableRecommendationList(Comparator<Recommendation> comparator) {
+        requireNonNull(comparator);
+        observableRecommendations.setComparator(comparator);
+    }
+
+    @Override
+    public void updateObservableRecommendationList(Predicate<Recommendation> predicate) {
+        requireNonNull(predicate);
+        filteredRecommendations.setPredicate(predicate);
+    }
```

Comment:
@zichen-3974 This is likely where you'll be able to get meets from recommendations.

Reply:
Nice! Noted and thanks.